Show End time in Autofill

Clashsoft opened the pull request:
- Autofill now shows the End time.
- Autofill modal now has the correct title.
- Improved Autofill modal layout.

Closes #61 

<img width="818" alt="Screenshot 2023-04-22 at 15 43 45" src="https://user-images.githubusercontent.com/4145923/233788475-498cff24-bc1b-4329-858f-75c0c9500b9f.png">

Morphclue commented on frontend/src/app/modals/autofill-modal/autofill-modal.component.html:
```diff
@@ -0,0 +1,78 @@
+    </form>
+  </ng-container>
+  <ng-container modal-footer>
+    <div class="text-muted bi-info-circle">
+      End time: {{ endTime }}
+    </div>
```
I think it would be useful to add a hint text in addition to the info-circle, which explains end time in more detail.
"The end time is calculated based on the start time you specify, the duration of the events, the pauses between events, and the number of repetitions."

Clashsoft replied:
Done!

Morphclue reviewed (changes requested):
Except from the one suggestion I have this looks good.

Clashsoft commented:
New changes:

- Added different hover color
- Added highlighting for today
- Added tooltip

![image](https://user-images.githubusercontent.com/4145923/233794858-a59a67c1-aea0-4fba-884b-38ab9c0db385.png)
